pom.xml

```diff
@@ -210,6 +210,16 @@
         <artifactId>javax.servlet.jsp.jstl-api</artifactId>
         <version>1.2.2</version>
       </dependency>
+	  <dependency>
+		 <groupId>javax.servlet</groupId>
+		 <artifactId>jstl</artifactId>
```

Comment:
taglibs depends on https://mvnrepository.com/artifact/org.apache.taglibs/taglibs-standard-spec/1.2.5 and this will conflict. Do you get any duplicate classloader warnings when you run the code?